client/my-sites/sidebar/sidebar.jsx

```diff
@@ -345,13 +347,23 @@ export class MySitesSidebar extends Component {
 	};
 
 	store() {
+		const allowedCountryCodes = [ 'US', 'CA' ];
```

Comment:
I also suggest moving this closer to the isCountryAllowed calculation instead of up here